deploy/deployment-manager/deploy_service_account.sh

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env bash
 set -e
 
-# this script:
-# 1. enables necessary APIs for CSPM GCP integration
-# 2. applies a Deployment Manager template to create a service account with role and a key
-# 3. saves generated key to a file (KEY_FILE.json) and prompts the user to copy-paste it to the GCP integration
-# 4. handles deployment failure cleanup
+# This script:
+# 1. Enables necessary APIs for CSPM GCP integration
+# 2. Applies a Deployment Manager template to create a service account with roles and key
+# 3. Saves generated key to KEY_FILE.json
+# 4. Handles retry for deployment and failure cleanup
```

Comment:
maybe include a comment about `dependsOn` not being sufficient

Reply:
Added an additional commit to clarify the `dependsOn` issue.